backend/app/schemas/AccesLogSchema.js

```diff
@@ -0,0 +1,10 @@
+const { z } = require('zod')
+
+const AccesLogSchema = z.object({
+  id: z.number().int().optional(),
```

Comment:
Make int required. User omit when it is not required: https://github.com/colinhacks/zod#pickomit